deploy/helm/grafana-operator/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.1
+version: 0.1.0-rc0
```

Comment:
This would force us to make changes like this before cutting a new release. But we will still need to make prepare PR:s just like we do in our current workflow so I don't think it's a huge problem.